src/main/java/Event.java

```diff
@@ -1,22 +1,28 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Event extends Task {
 
-    protected String from;
-    protected String to;
+    protected LocalDateTime fromDateTime;
```

Comment:
You can consider simplifying the naming to start and end to ease understanding of what the variables mean